src/Components/Base/Avatar/style.ts

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import { StyledAvatarProp, StyledImageProp } from './type';
+
+export const StyledAvatar = styled.div<StyledAvatarProp>`
+  position: relative;
+  display: inline-block;
+  overflow: hidden;
+
+  border: 1px solid
+    ${({ theme }) => `
+      ${theme.colors.border}
+    `};
+  border-radius: ${({ $shape }) =>
+    ({
+      circle: '50%',
+      round: '4px',
+      square: '0px',
+    })[$shape || 'square']};
```

Comment:
제가 테스트 해봤는데, 디폴트 size인 20에서는 괜찮은데 사이즈가 커질수록 4px가 너무 작게 느껴져서, 이걸 size에 따라 비율을 조절해주는게 좋을 것 같아요. 

```suggestion
  border-radius: ${({ $shape, $size }) =>
    ({
      circle: '50%',
      round: `${$size / 5}px`,
      square: '0px',
    })[$shape || 'square']};
```

```jsx
//index.ts
<StyledAvatar
      $shape={shape}
      $size={size}
    >
```

이런식으로? 비율은 임의로 size / 5 로 했습니당 ㅋㅋㅋㅋ

Reply:
피드백 감사합니다! 저도 몇 가지 단위 테스트 해본 후 반영해놓겠습니다 🙇🏻‍♂️